keyboards/thevankeyboards/minivan/keymaps/jetpacktuxedo/config.h

```diff
@@ -1,7 +1,9 @@
 #ifndef CONFIG_USER_H
```

Comment:
Actually, this entire file could be replaced with just: 

```c
#pragma once

#define PERMISSIVE_HOLD
#define RGBLIGHT_EFFECT_BREATHING
```

Reply:
done!